.github/workflows/main.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - master
       - stable
+  
+  # Allow this workflow to be triggered on the master branch by an external system (such as the kaitai_struct repo's workflow)
+  repository_dispatch:
```

Comment:
I'd specify an exact event type here as the [documentation suggests](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#repository_dispatch) (using `types`) so that it's easier to track (e.g. using [GitHub code search](https://github.com/search?q=org%3Akaitai-io%20&type=code)) that "this specific event type is triggered in [`kaitai_struct`](https://github.com/kaitai-io/kaitai_struct) and expected in [`kaitai_struct_webide`](https://github.com/kaitai-io/kaitai_struct_webide)".